### PR TITLE
[DE-7] Move allure vars to script

### PR DESCRIPTION
**Pull request by denisqsound**

### CODE REVIEW

File: test/docker-compose.single.yaml [hunk 1]
```diff
@@ -21,17 +21,17 @@ services:
       WALLARM_API_HOST: ${WALLARM_API_HOST}
       WALLARM_API_PRESET: ${WALLARM_API_PRESET:-audit}
       WALLARM_API_CA_VERIFY: ${WALLARM_API_CA_VERIFY}
-      CLIENT_ID: ${CLIENT_ID}
+      CLIENT_ID: ${CLIENT_ID:-5}
```
Comment (xDmitriev) [inline test/docker-compose.single.yaml]: Don't you want to put all of them in Makefile, as env vars? With the optional ability to overwrite from .env file.

Comment (denisqsound) [reply test/docker-compose.single.yaml]: Move to Makefile as env vars

File: Makefile [hunk 2]
```diff
@@ -14,6 +14,9 @@ WLRM_FOLDER       = stable-$(shell echo ${NGINX_VERSION} | sed 's/\.//g')
 GOMPLATE_VERISON  = 3.11.7
 COMMIT_SHA        ?= git-$(shell git rev-parse --short HEAD)
 
+ALLURE_ENDPOINT ?= https://allure.wallarm.com
```
Comment (xDmitriev) [inline Makefile]: please, place them below line 35 `.EXPORT_ALL_VARIABLES:`

Comment (denisqsound) [reply Makefile]: moved

**Review by xDmitriev — APPROVED** [review]

**Review by xDmitriev — APPROVED** [review]

**Pull request merged**